Makefile

```diff
@@ -3,6 +3,10 @@ VERSION ?= 1.1.1
 # Default bundle image tag
 IMAGE_TAG_BASE ?= controller
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+# Default Namespace
+NAMESPACE ?= openshift-storage
```

Comment:
Why do we need this? If I am not mistaken the resulted namespace is already customizable on `make bundle` run (it is a kustomize feature)

Reply:
We need this to add `namespace: openshift-storage` to the CSV. Currently when we run `make bundle` command the CSV has `namespace: placeholder` but I have used kustomize to add the `namespace: openshift-storage`(default) or we can pass NAMESPACE parameter with make bundle command to set it to something else. So yes this can be done using kustomize but we have to type it manually before running `make bundle` command but now it will run when `make bundle` command is executed.